stack.yaml

```diff
@@ -21,3 +21,4 @@ extra-deps:
 - yaml-0.8.17
 - nonce-1.0.2
 - persistent-2.5
+- cookie-0.4.2
```

Comment:
The [current `cookie` package constraint is `>= 0.4.1`](https://github.com/yesodweb/yesod/blob/465210d131ab7a82807e1a0fd2a02d3c0063d025/yesod-core/yesod-core.cabal#L45), but the SameSite options were added in 0.4.2. Can you bump the `cookie` constraint in the cabal file to handle this?